src/applications/disability-benefits/526EZ/tests/containers/ConfirmationPage.unit.spec.jsx

```diff
@@ -21,10 +21,14 @@ describe('Disability Benefits 526EZ <ConfirmationPage>', () => {
       <ConfirmationPage form={form}/>
     );
 
-    expect(tree.find('.confirmation-page-title').text()).to.equal('Claim submitted');
-    expect(tree.find('span').at(1).text().trim()).to.equal('For Sally  Alphonse');
-    expect(tree.find('p').at(0).text()).to.contain('We usually process claims within 103 days.');
-    expect(tree.find('p').at(1).text()).to.contain('We may contact you if we have questions or need more information. You can print this page for your records.');
-    expect(tree.find('.disability-list').text()).to.contain('PTSD');
+    expect(tree.find('.confirmation-page-title').render().text()).to.equal('Claim submitted');
+    expect(tree.find('span').at(1)
+      .render()
```

Comment:
Idle curiosity: Why break this one out onto separate lines?

Reply:
Hah, ask the linter 😛

Reply:
:roll_eyes: